add docs for v2 #21 (#24)
code-rhapsodie/dataflow-bundle: README.md

* Enable/Disable a scheduled Dataflow from the command line
* Display the list of scheduled Dataflow from the command line
* Display the result for the last Job for a Dataflow from the command line

> * Work with many Doctrine DBAL connection

Reviewer: ```suggestion
* Work with multiple Doctrine DBAL connections
```

### Update the database

> This bundle uses Doctrine DBAL for drive the database table for store Dataflow schedule (`cr_dataflow_scheduled`)

Reviewer: ```suggestion
This bundle uses Doctrine DBAL to store Dataflow schedule into the database table (`cr_dataflow_scheduled`)
```

and jobs (`cr_dataflow_job`).

If you use [Doctrine Migration Bundle](https://symfony.com/doc/master/bundles/DoctrineMigrationsBundle/index.html) or [Phinx](https://phinx.org/) 
or [Kaliop Migration Bundle](https://github.com/kaliop-uk/ezmigrationbundle) or whatever,
you can add a new migration with the generated SQL query from this command:

```shell script
$ bin/console code-rhapsodie:dataflow:dump-schema
```

If you have already the tables, you can add a new migration with the generated update SQL query from this command:

```shell script
$ bin/console code-rhapsodie:dataflow:dump-schema --update
```

## Configuration

> By default the Doctrine DBAL connection used is `default`. You can change the default connexion with the configuration.

Reviewer: ```suggestion
By default, the Doctrine DBAL connection used is `default`. You can configure the default connection.
```

`code-rhapsodie:dataflow:execute` Let you execute one dataflow job.

`code-rhapsodie:dataflow:dump-schema` Generates schema create / update SQL queries

### Work with many databases

> All commands have `--connection` option to define what Doctrine DBAL use to execution. You must be register in your 

Reviewer: ```suggestion
All commands have a `--connection` option to define what Doctrine DBAL connection to use during execution.
```

> crontab one job by connection.

Reviewer: ```suggestion
```

> Exemple:

Reviewer: ```suggestion
Example:
```

> This command use the `default` DBAL connection to generates all update schema update queries.

Reviewer: ```suggestion
This command uses the `default` DBAL connection to generate all schema update queries.
```

```shell script
$ bin/console code-rhapsodie:dataflow:dump-schema --update --connection=default
```

> To execute all pending job in tow specific connection use:

Reviewer: ```suggestion
To execute all pending job for a specific connection use:
```
